src/test/java/database/service/JsonDatabaseServiceTest.java

```diff
@@ -106,6 +107,18 @@ void updateRecordInTableWIthCorrectIdTest() {
         assertEquals(secondStudent, jsonDatabaseService.getById(Student.class, 1));
     }
 
+    @Test
+    void updateOxfordStudent() {
+        jsonDatabaseService.createTable(OxfordStudent.class);
+        OxfordStudent os = new OxfordStudent("N", 4.5, 20);
+        OxfordStudent os2 = new OxfordStudent("M", 4.2, 21);
+
+        jsonDatabaseService.addNewRecordToTable(os);
```

Comment:
Please check that student created and has id=0.

Reply:
Discussed offline. Decided to do not add extra check.